packages/cli/src/commands/generate/sdl/__tests__/fixtures/enumGeneratedSdl.js

```diff
@@ -0,0 +1,35 @@
+export const schema = gql`
+  type Shoe {
+    id: Int!
+    color: Color!
+  }
+
+  enum Color {
+
+    RED
+
+    GREEN
+
+    BLUE
+
+  }
```

Comment:
I'm not sure if you ended up fixing the generation code, but the tests definitely need to be updated to ditch the extra lines.

```suggestion
  enum Color {
    RED
    GREEN
    BLUE
  }
```

Reply:
Both tests and `template` updated.